api/app/signals/apps/users/factories.py

```diff
@@ -51,3 +53,18 @@ class Meta:
     name = factory.LazyAttribute(
         lambda o: fake.word()
     )
+
+
+class UserProfileFactory(factory.DjangoModelFactory):
```

Comment:
`UserProfileFactory` is not used anywhere and not really needed, so it could be removed.